src/arrayMethodSort.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    const defaultCompare = (el1, el2) => {
+      const itemOne = `${el1}`;
+      const itemTwo = `${el2}`;
+
+      return itemOne > itemTwo ? +1 : -1;
```

Comment:
The default compare function should return 0 when the two elements are equal to maintain consistency with the native sort method. As it stands, it will never return 0 which might lead to an incorrect sort order in some cases.